.travis.yml

```diff
@@ -1,10 +1,16 @@
 language: node_js
 cache: npm
+dist: bionic
 stages:
   - check
   - test
   - cov
 
+branches:
+  only:
+  - master
+  - /^release\/.*$/
+
```

Comment:
Won't this  disable tests on PRs?

Reply:
No - see the checks that have run on this PR.

If you don't specify this they run twice - once for the PR and once for the branch.

Reply:
Ah, Thanks! I  forgot this is Travis, not GoogleActions :man_facepalming: